### PR TITLE
added info for some components

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added info for image, input and list components

### CODE REVIEW

File: input.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            </h3>
+        </header>
+        <section class="page-section">
+            <p class="paragraph1 page-section--para">
+                Input components are provided for text and text-area inputs.
+            </p>
+            <h4 class="h4 page-section--title">Required Input</h4>
+            <div class="page-section--code control-display">
+                <div class="input">
+                    <label class="input-label" for="name">Full Name*</label>
```
Comment (itsmebipul00) [inline input.html:91]: id in input

Comment (DivyanshSareen) [reply input.html:91]: it is there in the input element, the div is just a wrapper

Comment (itsmebipul00) [reply input.html:91]: ohh my bad

**Review by itsmebipul00 — APPROVED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

**Pull request merged**